### ISSUE TITLE
Quick check to find the reason for aa_mismatch QC issues

### ISSUE THREAD

**Issue by ReneRanzinger** [glygener/glygen-issues#1671]
Based on #1641. Look at some examples for this case:
```
80256,aa_mismatch,human_proteoform_glycosylation_sites_pdc_ccrcc
```
Lets try to find out what happened. If the problem is that the glycan annotation was done on an isoform and it failed since Robel assumes the canonical it means we need to get the isoform2canonical mapper into the pipeline. If there is another case and the error is on their side we should try to contact the dataset owners and either get feedback from them or ask them to fix the issue. 

If the authors can or do not want to resolve the issue we need to discuss in a group meeting if we want to treat them as fuzzy annotation (glycan somewhere on the protein, no position or range).

**Comment by katewarner** [comment]
@ReneRanzinger @rykahsay @jeet-vora 

I checked a few of the aa_mismatches in `human_proteoform_glycosylation_sites_pdc_ccrcc` and looked at the original ticket for creating the dataset. 

From the ones I checked, it looks like the error is on their side. The peptides that I checked (see below) do align with the canonical sequence but the glycosylation aa they provided to us are incorrect. These glycosylation sites were taken directly from their results and I can't work out from their methods in their paper how they identified their glycosylation aa. I will try to contact the authors to ask them about it, once I've looked at the other global QC flags from this dataset. 

**Matches canonical but glycosite is 281:**
"Q7Z7G0-1","288","Asn","G90659AW","N-linked","protein_xref_pubmed","37074911","protein_xref_pdc","PDC000471","N4H5F1S0G0","","UBERON:0002113","kidney","288","288","N","N","APLGGVILVHLIIPGLnETTVK","known_site","PAS","","CPT0026410003","aa_mismatch"

**Matches canonical but glycosite is 337:**
"P08575-3","335","Asn","G70441OD","N-linked","protein_xref_pubmed","37074911","protein_xref_pdc","PDC000471","N5H6F1S0G0","","UBERON:0002113","kidney","335","335","N","N","NIETFTCDTQnITYR","known_site","TQN","","CPT0075170001","aa_mismatch"

**Matches canonical but glycosite is 1291:**
"P02751-15","7","Asn","G37509XX","N-linked","protein_xref_pubmed","37074911","protein_xref_pdc","PDC000471","N6H4F1S0G0","","UBERON:0002113","kidney","7","7","N","N","WTPLnSSTIIGYR","known_site","PGL","","CPT0012550003","aa_mismatch"

**Comment by ReneRanzinger** [comment]
- [x] Compile a list of the errors and report them to the dataset owners. We should keep the email open and offer our help in fixing that. But primarily that should be their responsibility. 
- [x] Start a new document in Sharepoint to record this work. It should show how many errors we found. Who we reported them to. Did they act on this. How many were fixed. This can be used for the annual progress report to show impact of GlyGen on other resources.
